web/src/services/posthogClient.ts

```diff
@@ -0,0 +1,43 @@
+import { PostHog } from "posthog-node";
```

Comment:
let's add clear documentation that this is for server-side use only. perhaps add a directive to prevent accidental use from the client side.